### PR TITLE
Update description for `top` slot

### PR DESCRIPTION
**Pull request by tayalav**
## Launch Checklist

Updated description for the "top" slot based on recent changes

### CODE REVIEW

File: MIGRATION_GUIDE_v3.md [hunk 1]
```diff
@@ -117,9 +117,10 @@ Slot | Description
 --- | ---
 `bottom` | Above polygons (land, landuse, water, etc.)
 `middle` | Above lines (roads, etc.) and behind 3D buildings
-`top` | Above all existing layers in the style
+`top` | Above POI labels and behind Place and Transit labels
 
 Set the preferred `slot` on the `Layer` object before adding it to your map and your layer will be appropriately placed in the Standard style's layer stack.
+If no slot is specified for a custom layer then layers are still placed at the very top of the layer list, providing a location that has highest collision priority compared to all other layers.
```
Comment (jcritchley) [inline MIGRATION_GUIDE_v3.md:123]: @tayalav we could maybe shorten this sentence? 

> If no slot is specified for a custom layer they are placed at the very top of the layer list, providing a location that has highest collision priority compared to all other layers.

**Review by jcritchley — COMMENTED** [review]
@tayalav thanks for doing this! I had one suggestion for wording and looks like you might need to also update the changelog?

File: MIGRATION_GUIDE_v3.md (hunk 1 above)
Comment (tayalav) [reply MIGRATION_GUIDE_v3.md]: @jcritchley sure, thanks fro the suggestion!

**Review by jcritchley — APPROVED** [review]
Looks good!

File: CHANGELOG.md [hunk 2]
```diff
@@ -13,6 +13,7 @@ Mapbox GL JS v3 enables the Mapbox Standard Style, a new realistic 3D lighting s
 
 ### Bug fixes 🐞
 
+- Update the `top` slot description in the docs.
```
Comment (stepankuzmin) [inline CHANGELOG.md:16]: The changelog is updated separately, so you don't have to update it in this PR.

```suggestion
```

**Review by stepankuzmin — APPROVED** [review]
Just one nit 👍

**Pull request merged**